Makefile

```
@@ -155,7 +151,11 @@ ghcid:
 
 # Used by CI
 .PHONY: lint-all
-lint-all: formatc hlint-check-all check-local-nix-derivations treefmt-check
+lint-all: formatc hlint-check-all check-local-nix-derivations treefmt-check # weeder (does not work on CI yet)
+
+.PHONY: weeder
+weeder:
+	time weeder -N || echo -e '\n\n*** make sure you have added this to your cabal.project.local and make clean before building wire-server:\n\npackage *\n  ghc-options: -fwrite-ide-info\n\n'
```

```suggestion
	weeder -N || echo -e '\n\n*** make sure you have added this to your cabal.project.local and make clean before building wire-server:\n\npackage *\n  ghc-options: -fwrite-ide-info\n\n'
```

Also it seems like we're gonna print this message on any failure, my guess is that it will definitely be confusing in future.